docs/resources/integration_gcp.md

```diff
@@ -8,7 +8,7 @@ description: |-
 
 # datadog_integration_gcp (Resource)
 
-This resource is deprecated — use the `datadog_integration_gcp_sts resource` instead. Provides a Datadog - Google Cloud Platform integration resource. This can be used to create and manage Datadog - Google Cloud Platform integration.
+This resource is deprecated — use the `datadog_integration_gcp_sts` resource instead. Provides a Datadog - Google Cloud Platform integration resource. This can be used to create and manage Datadog - Google Cloud Platform integration.
```

Comment:
```suggestion
This resource is deprecated—use the `datadog_integration_gcp_sts` resource instead. Provides a Datadog - Google Cloud Platform integration resource. This can be used to create and manage Datadog - Google Cloud Platform integration.
```

Reply:
@brett0000FF per line 6 should we just update both to stay consistent?